examples/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ add_subdirectory(ilu-preconditioned-solver)
 add_subdirectory(inverse-iteration)
 add_subdirectory(iterative-refinement)
 add_subdirectory(minimal-cuda-solver)
+add_subdirectory(multiprecision-ir)
```

Comment:
I know that this is a bigger change. But let's stick to the notation "mixed precision iterative refinement". That is well established - along with the abbreviation MPIR.

Reply:
Can you change the subdirectory name?